dependencies.props

```diff
@@ -1,14 +1,76 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Source of truth for dependency tooling: the commit hash of the dotnet/versions master branch as of the last auto-upgrade. -->
+
+  <!-- For detailed explanation of the reference versions see:
+
+        https://github.com/dotnet/buildtools/blob/master/Documentation/project-nuget-dependencies.md
+        https://github.com/dotnet/buildtools/blob/master/Documentation/annotated-dependency-props.md
+
+       These ref versions are pulled from https://github.com/dotnet/versions.
+  -->
   <PropertyGroup>
-       <CoreCLRVersion>2.0.0-preview2-25226-02</CoreCLRVersion>
-       <JitVersion>2.0.0-preview2-25226-02</JitVersion>
+    <CoreFxCurrentRef>824797963a7c7cfce78899f9c1af01e1011a1bd9</CoreFxCurrentRef>
+    <CoreClrCurrentRef>824797963a7c7cfce78899f9c1af01e1011a1bd9</CoreClrCurrentRef>
+    <StandardCurrentRef>824797963a7c7cfce78899f9c1af01e1011a1bd9</StandardCurrentRef>
+  </PropertyGroup>
+  
+  <!-- Auto-upgraded properties for each build info dependency. -->
+  <PropertyGroup>
+    <CoreFxExpectedPrerelease>preview2-25301-02</CoreFxExpectedPrerelease>
+    <CoreClrExpectedPrerelease>preview2-25301-02</CoreClrExpectedPrerelease>
+    <StandardExpectedPrerelease>preview2-25301-01</StandardExpectedPrerelease>
```

Comment:
Removing these would be great, so that nobody down the line can try `2.0.0-$(CoreFxExpectedPrerelease)`. It makes the repo fragile to version revving down the road.